crates/openapi/src/routes/payments.rs

```diff
@@ -629,3 +629,23 @@ pub fn payments_post_session_tokens() {}
 )]
 #[cfg(feature = "v2")]
 pub fn payments_create_intent() {}
+
+/// Payments - Get Intent
+///
+/// **Get a payment intent object when id is passed in path**
+///
+/// You will require the 'API - Key' from the Hyperswitch dashboard to make the call.
+#[utoipa::path(
+  get,
+  path = "/v2/payments/{id}/get-intent",
+  params (("id" = String, Path, description = "The unique identifier for the Payment Intent")),
+  responses(
+      (status = 200, description = "Payment Intent", body = PaymentsIntentResponse),
+      (status = 404, description = "Payment ID not found")
```

Comment:
Should this be `400` instead? What all status codes need to be mentioned here?

Reply:
This will be 404 only, for now other status code not needed to be mentioned.